lib/server-starter.js

```diff
@@ -40,15 +40,18 @@ export default class ServerStarter extends EventEmitter {
    * @param {string} cmd - Server command to run.
    * @param {string[]} args - Arguments to use with server command.
    * @param {object} [options] - Optional settings.
+   * @param {object} [options.env=process.env] - Environment key-value pairs.
    * @param {boolean} [options.stdout=false] - Forward server output from STDOUT to STDOUT.
    * @param {boolean} [options.stderr=true] - Forward server output from STDERR to STDERR.
    */
   launch(cmd, args, options = {}) {
     if (typeof this._process !== 'undefined') throw new Error('Server already launched');
     const stdout = typeof options.stdout !== 'undefined' ? options.stdout : false;
     const stderr = typeof options.stderr !== 'undefined' ? options.stderr : true;
+    const spawn_options = {stdio: ['pipe', 'pipe', 'pipe', this._fd]};
+    if (typeof options.env !== 'undefined') spawn_options.env = options.env;
```

Comment:
`if (options.env !== undefined)` seems more efficient.

Reply:
I copied the pattern from the above instances. Should I update those as well?

Reply:
That would be nice. Some of this code needs a bit of cleanup. 🤔